### ISSUE TITLE
Enhance IPv4 operators to accept non-constant arguments

### ISSUE THREAD

**Issue by mrosi** [questdb/questdb#4839]
### Is your feature request related to a problem?

Some IPv4 operators like `<<` and function `netmask` accept only `string` constants, so that there is very limited usage with table data.

### Describe the solution you'd like.

All IPv4 operators and functions should accept 'non-constant' values taken from DB Table.
Simplified usecase:
```sql
select
  ipaddr, subnet,
  netmask('10.16.0.0/12') mask_string, ipaddr << '10.16.0.0/12' is_in_string,
  netmask(subnet) mask_value, ipaddr << subnet is_in_value
from (select ipv4 '10.16.0.11' ipaddr,'10.16.0.0/12' subnet);
```
This feature is mandatory for usage in `where` and `join` clauses

### Full Name:

Martin Rosicky

### Affiliation:

Simac/Vodafone

**Comment by mrosi** [comment]
Hi,
do you have any idea when this feature could be implemented? Unavailability of full support of IPv4/IPv6 operations is going to be showstopper for QuestDB in analysis of network traffic.
Thanks a lot for your support
Martin R.

**Comment by puzpuzpuz** [comment]
@mrosi I'm working on a patch for this one. It should be included in the next release.

**Comment by puzpuzpuz** [comment]
@mrosi good news. The patch you wanted is released in v8.1.1. Please give it a try and let me know if something is missing.

**Comment by mrosi** [comment]
@puzpuzpuz, it works
thanks a lot
